### DISCUSSION TITLE
自定义模型剩余字数为什么会没有显示？

### DISCUSSION THREAD

**Discussion by 5478-lfy** [lobehub/lobehub#1230] [General Question] [answered]
### 💻 系统环境

Windows

### 📦 部署环境

Official Preview

### 🌐 浏览器

Chrome

### 🐛 问题描述

![image](https://github.com/lobehub/lobe-chat/assets/92673213/5fe89ba6-f80f-4c92-b1d4-8afcf62fb79c)
 我是按照名字匹配添加的模型，userchat=pgt-4,可以使用但是剩余字数会没有显示

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
`userchat` 没有最大上 token 的信息，自然无法显示了

[a comment by an automated account is not shown]

**Comment by buwanyuanshen** [comment]
你需要去文件里面修改添加模型设置字数，设置字数，大概有4个文件需要修改好像是😂，我改过，不过这玩意或许不是很重要。

**Reply by arvinxx** [reply]
现在只要改一处就好了 https://github.com/lobehub/lobe-chat/blob/main/src/config/modelProviders/openai.ts#L11
